cmd/createcluster.go

```diff
@@ -230,7 +230,13 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		return err
 	}
 
-	depositDatas, err := createDepositDatas(def.WithdrawalAddresses(), network, secrets)
+	depositAmounts := deposit.EthsToGweis(conf.DepositAmounts)
```

Comment:
`conf.DepositAmounts` will be nil if `conf.DefFile` is set. I suggest to set the `conf.DepositAmounts` from def object if `conf.DefFile` is provided considering def is of v1.8.0 version.

Reply:
Reworked.